From c980beff9a5d28548eaf644cf1a660960755249a Mon Sep 17 00:00:00 2001
From: Ian Fleming <iflema@yahoo.com.au>
Date: Thu, 20 May 2010 16:16:04 +0100
Subject: [PATCH] dpkg-architecture(1): Fix typos

DEB_HOST_ARCH_BITS was listed as DEB_BUILD_ARCH_BITS;
DEB_HOST_ARCH_ENDIAN was listed as DEB_BUILD_ARCH_ENDIAN.  In the
process, adjust the ordering to match dpkg-architecture's output.
Originally reported as
https://bugs.launchpad.net/ubuntu/+source/dpkg/+bug/564308.

---
 man/dpkg-architecture.1 |   14 +++++++-------
 1 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/man/dpkg-architecture.1 b/man/dpkg-architecture.1
index fdc0d5c..4ca309e 100644
--- a/man/dpkg-architecture.1
+++ b/man/dpkg-architecture.1
@@ -109,28 +109,28 @@ The Debian cpu name of the build machine.
 The pointer size of the build machine (in bits).
 .IP "\s-1DEB_BUILD_ARCH_ENDIAN\s0" 4
 The endianness of the build machine (little / big).
-.IP "\s-1DEB_BUILD_GNU_TYPE\s0" 4
-The \s-1GNU\s0 system type of the build machine.
 .IP "\s-1DEB_BUILD_GNU_CPU\s0" 4
-The \s-1CPU\s0 part of \s-1DEB_BUILD_GNU_TYPE\s0
+The \s-1CPU\s0 part of \s-1DEB_BUILD_GNU_TYPE\s0.
 .IP "\s-1DEB_BUILD_GNU_SYSTEM\s0" 4
 The System part of \s-1DEB_BUILD_GNU_TYPE\s0.
+.IP "\s-1DEB_BUILD_GNU_TYPE\s0" 4
+The \s-1GNU\s0 system type of the build machine.
 .IP "\s-1DEB_HOST_ARCH\s0" 4
 The Debian architecture of the host machine.
 .IP "\s-1DEB_HOST_ARCH_OS\s0" 4
 The Debian system name of the host machine.
 .IP "\s-1DEB_HOST_ARCH_CPU\s0" 4
 The Debian cpu name of the host machine.
-.IP "\s-1DEB_BUILD_ARCH_BITS\s0" 4
+.IP "\s-1DEB_HOST_ARCH_BITS\s0" 4
 The pointer size of the host machine (in bits).
-.IP "\s-1DEB_BUILD_ARCH_ENDIAN\s0" 4
+.IP "\s-1DEB_HOST_ARCH_ENDIAN\s0" 4
 The endianness of the host machine (little / big).
-.IP "\s-1DEB_HOST_GNU_TYPE\s0" 4
-The \s-1GNU\s0 system type of the host machine.
 .IP "\s-1DEB_HOST_GNU_CPU\s0" 4
 The \s-1CPU\s0 part of \s-1DEB_HOST_GNU_TYPE\s0.
 .IP "\s-1DEB_HOST_GNU_SYSTEM\s0" 4
 The System part of \s-1DEB_HOST_GNU_TYPE\s0.
+.IP "\s-1DEB_HOST_GNU_TYPE\s0" 4
+The \s-1GNU\s0 system type of the host machine.
 .
 .SH "DEBIAN/RULES"
 The environment variables set by \fBdpkg\-architecture\fP are passed to
-- 
1.7.1

